src/.env.example

```
@@ -1,4 +1 @@
-BLOCKFROST_SANCHONET_API=
-BLOCKFROST_PREVIEW_API=
-BLOCKFROST_PREPROD_API=
-BLOCKFROST_MAINNET_API=
\ No newline at end of file
+NEXT_PUBLIC_REST_IPFS_GATEWAY=
```

```suggestion
NEXT_PUBLIC_REST_IPFS_GATEWAY=ipfs.io/ipfs/
```

can we just have this value, because the value is hard to guess